### PR TITLE
Add ms_entropy

### PR DESCRIPTION
**Pull request by n-elie**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by n-elie** [comment]
@conda-forge-admin please rerender

File: recipes/ms_entropy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools >=61.0
+    - cython >=0.26.1
+    - numpy >=1.9.3
```
Comment (xhochy) [inline recipes/ms_entropy/meta.yaml:26]: Leave this unpinned to get the global pin
```suggestion
    - numpy
```

File: recipes/ms_entropy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```
Comment (xhochy) [inline recipes/ms_entropy/meta.yaml:30]: This should no longer be used, numpy has a run_exports now
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

File: recipes/ms_entropy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools >=61.0
+    - cython >=0.26.1
+    - numpy
+    - pip
+  run:
+    - python
+    - numpy
```
Comment (xhochy) [inline recipes/ms_entropy/meta.yaml:30]: Don't list it at all here. Still, this probably won't fix the build error.
```suggestion
```

**Comment by xhochy** [comment]
The patch looks empty. This is probably not intended.

**Pull request merged**